docs/src/main/asciidoc/amqp.adoc

```diff
@@ -20,22 +22,32 @@ To complete this guide, you need:
 * an IDE
 * JDK 11+ installed with `JAVA_HOME` configured appropriately
 * Apache Maven {maven-version}
-* A running AMQP 1.0 broker, or Docker Compose to start a development cluster
+* docker and docker-compose
 * GraalVM installed if you want to run in native mode.
 
 == Architecture
 
-In this guide, we are going to generate (random) prices in one component.
-These prices are written in an AMQP queue (`prices`).
-A second component reads from the `prices` queue and apply some magic conversion to the price.
-The result is sent to an in-memory stream consumed by a JAX-RS resource.
-The data is sent to a browser using server-sent events.
+In this guide, we are going to develop two applications communicating with an AMQP broker.
+We will use https://activemq.apache.org/components/artemis/[Artemis], but you can use any AMQP 1.0 broker.
+The first application sends a _quote request_ to an AMQP queue and consumes messages from the _quote_ queue.
+The second application receives the _quote request_ and sends a _quote_ back.
+
+image::amqp-qs-architecture.png[alt=Architecture, align=center,width=80%]
+
+The first application, the `producer`, will let the user request some quotes over an HTTP endpoint.
+For each quote request a random identifier is generated and returned to the user, to put the quote request on _pending_.
```

Comment:
Detail but I had a hard time parsing the beginning of the sentence :)
```suggestion
For each quote request, a random identifier is generated and returned to the user, to put the quote request on _pending_.
```

Reply:
Fixed.